### ISSUE TITLE
login options

### ISSUE THREAD

**Issue by drmrbrewer** [githubsaturn/captainduckduck#215]
I'd like to be able to `captainduckduck login` as part of a script I run to set up a new server.  Is it possible to supply command-line options to `login` to specify all the required information without prompting?  If not, would that be something that could be added in future?

**Comment by githubsaturn** [comment]
Deploy command takes in everything you need as parameters - in that case you don't need to run login. Instead of login command, you should use deploy.

**Comment by drmrbrewer** [comment]
Yes but I usually deploy manually via the command line, so it's not convenient to have to supply all the credentials at deploy time.  At some point, a login will be required, and it would be nice if at least the login could be scripted at server/container initialisation.

Anyway, as mentioned above, in the case where I'm putting cdd inside another container, I'm specifying the .config folder as a persistent directory, so cdd's state is persisted.  This is working nicely in that context.

**Comment by githubsaturn** [comment]
> Anyway, as mentioned above, in the case where I'm putting cdd inside another container, I'm specifying the .config folder as a persistent directory, so cdd's state is persisted. This is working nicely in that context.

The issue with this approach is that, whenever your server gets rebooted, you have to manually execute login command again. I'll leave this issue open as I think adding the ability to define login params is nice anyway.
